Tests/Test/WebTestCaseTest.php

```diff
@@ -324,9 +323,6 @@ public function testLoadFixturesFilesPaths()
         );
     }
 
-    /**
-     * @QueryCount(100)
```

Comment:
Now yet, I tried but I get an error:

> 1) Liip\FunctionalTestBundle\Tests\Test\WebTestCaseConfigTest::testAdminAuthenticationLoginAs
> Doctrine\Common\Annotations\AnnotationException: [Semantical Error] The annotation "@Liip\FunctionalTestBundle\Annotations\QueryCount" in method Liip\FunctionalTestBundle\Tests\Test\WebTestCaseConfigTest::testAdminAuthenticationLoginAs() does not exist, or could not be auto-loaded.

It looks like the annotation reader break some things, I have errors with PHPUnit's annotations too:

> 1) Liip\FunctionalTestBundle\Tests\Test\WebTestCaseConfigTest::testAdminAuthenticationInMemoryRoleAdmin
> Doctrine\Common\Annotations\AnnotationException: [Semantical Error] The annotation "@depends" in method Liip\FunctionalTestBundle\Tests\Test\WebTestCaseConfigTest::testAdminAuthenticationInMemoryRoleAdmin() was never imported. Did you maybe forget to add a "use" statement for this annotation?